zulip_bots/zulip_bots/lib.py

```diff
@@ -159,6 +161,48 @@ def state(self, default):
         yield new_state
         self.set_state(new_state)
 
+def setup_default_commands(bot_details, message_handler):
+    def def_about():
+        if bot_details['description'] == "":
+            return "**{}**".format(bot_details['name'])
+        return "**{}**: {}".format(bot_details['name'], bot_details['description'])
```

Comment:
This doesn't feel particularly readable.  A better approach might be to write these as as e.g.

`"**{name}**: {description}".format(**bot_details)`

and probably even better would be e.g.

`"**%(name)s**: %(description)s" % bot_details